src/pages/aboutYourself/AboutForm.jsx

```diff
@@ -22,13 +22,26 @@ const AboutForm = () => {
   const userLoading = useSelector(selectUserLoading);
   const userError = useSelector(selectUserError);
 
-  const [bio, setBio] = useState('');
+  const [bio, setBio] = useState('' || localStorage.getItem('aboutBio'));
 
-  const [socialTags, setSocialTags] = useState([]);
+  const [socialTags, setSocialTags] = useState(
+    [] || localStorage.getItem('social')
+  );
   const [professionalTags, setProfessionalTags] = useState([]);
   const [dietaryTags, setDietaryTags] = useState([]);
   const [cuisineTags, setCuisineTags] = useState([]);
 
+  const [minTags, setMinTags] = useState({
+    Social: { minimum: 10, show: false, numClicked: 0 },
+    Professional: { minimum: 3, show: false, numClicked: 0 },
```

Comment:
These are seeded  @ 1 min professional...can we shift this off to a constant so it's easy to change as needed?